_data/simple-icons.json

```diff
@@ -7266,8 +7266,8 @@
         },
         {
             "title": "Immich",
-            "hex": "4250AF",
```

Comment:
Where are you getting this new color from? I'm still seeing `#4250AF` as the primary color on their website (in light mode).

Reply:
Got the color straight from the official SVG here: https://github.com/immich-app/immich/blob/25c9b779e4d19bc34551f8b137266a459e0d70e1/design/immich-logo-stacked-light.svg


<img width="377" alt="Screenshot 2024-03-29 at 21 05 05" src="https://github.com/simple-icons/simple-icons/assets/47571755/a82d9792-4353-4401-852c-3d0e7ea889e6">

Reply:
That's a multi-colored SVG though. I think it's best to stick with their primary color on the website.

Reply:
Reverted to use 4250AF